README.md

```diff
@@ -1 +1,8 @@
-# docker-builder
\ No newline at end of file
+# Aeternity builder image
+
+Includes:
```

Comment:
May I suggest removing what it includes - that is a duplicated of Dockefile itself - and include use cases? (It is not trivial: both builder for epoch and image for Circle CI in epoch repo.)

Reply:
changed in 91caf1a